assets/src/store.rs

```diff
@@ -0,0 +1,95 @@
+use std::error::Error;
+use std::fs::File;
+use std::io::Error as IoError;
+use std::path::PathBuf;
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+/// An `Allocator`, holding a counter for producing unique ideas for the stores.
```

Comment:
:smile: :smile: :smile: Err, this somehow happens to me in the last time - I speak what I'm about to write in my head but write words that sound similar instead.